docs/manual/_quarto.yml

```diff
@@ -8,12 +8,29 @@ book:
     - index.qmd
     - install.qmd
     - getting-started.qmd
-    - cli.qmd
-    - api.qmd
+    - cli-reference.qmd
```

Comment:
Maybe move this and next one within CLI and API sections?

Reply:
I wasn't able to add sub-sections in any reasonable way, I tried, but it's like this with the "book" project type it seems. I can use "parts", but it changes the whole structure of the book into something we don't want. The only other alternative would be to join them back together again, to have reference and usage in the same file, but I think this way is best since I need to have the api-reference separate in any case (due to it being generated as such). What do you think?

Reply:
Yeah, I see what you mean. Ok, we can leave it as it is.